### PR TITLE
Nav

### PR DESCRIPTION
**Pull request by nikkypyra**

### CODE REVIEW

File: src/pages/TaskForm.js [hunk 1]
```diff
@@ -39,65 +38,66 @@ export default function TaskForm({ pets, addTask }) {
...
+                value={person}
+                maxLength="100"
+                placeholder="Insert person to complete task"
+                onChange={(e) => setPerson(e.target.value)}
+                required
+              />
+            </label>
+          </div>
+          <SubmitButton text="Submit" />
+          <p>*Mandatory Fields</p>
```
Comment (uwejettkant) [inline src/pages/TaskForm.js:98]: Are the  * typos or do they be intended?

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import { NavLink } from 'react-router-dom'
+import { ReactComponent as Home } from '../icons/home.svg'
```
Comment (uwejettkant) [inline src/components/Navigation.js:4]: Is it the required way of importing components for SVG's? Wouldn`t "import Home from '../icons/home.svg' work as well?

**Review by uwejettkant — COMMENTED** [review]
Looks good. Well done.

File: src/pages/TaskForm.js (hunk 1 above)
Comment (nikkypyra) [reply src/pages/TaskForm.js:98]: they are intended :)

File: src/components/Navigation.js (hunk 2 above)
Comment (nikkypyra) [reply src/components/Navigation.js:4]: I think that would work as well. This way had the benefit that it was instantly a component that I could style.

**Pull request merged**